tests/integration/__init__.py

```diff
@@ -0,0 +1,12 @@
+def get_azure_spark_conf():
```

Comment:
this must not be in this file, because everything in the integration testing must be almost real. this is dummy.

why it's not in conftest.py?